ios/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 6.0.0
+version: 6.1.1
```

Comment:
Why the bump here? I see there is #105 which bumps the version already.

Reply:
You are right. No need. We can release this PR with #105 .